### PR TITLE
Add note about support queries to contributing guidelines

### PR DESCRIPTION
**Pull request by bennuttall**
@JamesH65 ok with this? If so, it can go to copy edit.

### CODE REVIEW

**Comment by JamesH65** [comment]
Looks good to me, pushing to copy edit.

**Comment by bennuttall** [comment]
Pushed another update. Can we get it copy edited? @JaninaPi

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 # Contributing
 
-Please observe our contribution guidelines before creating an issue or pull request.
+Please reard our contribution guidelines before creating an issue or pull request.
```
Comment (JamesH65) [inline CONTRIBUTING.md]: Typo!

**Comment by JamesH65** [comment]
Looks OK to me.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bennuttall) [reply CONTRIBUTING.md]: Making work for yourself :P

**Pull request merged**